src/react-a11y-dialog.tsx

```diff
@@ -88,6 +88,9 @@ export const useA11yDialog = (props: ReactA11yDialogProps) => {
   ] as [A11yDialogLib | null, Attributes]
 }
 
+// Re-export the `a11y-dialog` instance type for convenient typing of useRef/dialogRef.
+export type A11yDialogInstance = A11yDialogLib
```

Comment:
Wondering whether we should export it as `A11yDialog`. What do you think?

Reply:
Sure, either is fine by me, and I'm sure you have a better idea about the most precise naming.

Reply:
Pushed a new commit.